src/fabric/src/fabric2_active_tasks.erl

```diff
@@ -0,0 +1,35 @@
+-module(fabric2_active_tasks).
+
+
+-export([
+    get_active_tasks/0,
+    get_active_task_info/1,
+
+    update_active_task_info/2
+]).
+
+
+-define(ACTIVE_TASK_INFO, <<"active_task_info">>).
+
+
+get_active_tasks() ->
+    couch_jobs_fdb:tx(couch_jobs_fdb:get_jtx(undefined), fun(JTx) ->
+        Types = couch_jobs:get_types(JTx),
+        lists:foldl(fun(Type, TaskAcc) ->
+            JobIds = couch_jobs:get_active_jobs_ids(JTx, Type),
+            Tasks = lists:map(fun(JobId) ->
```

Comment:
For jobs that don't have an ?ACTIVE_TASK_INFO field, we might want to not return an empty object but just skip it from the result list altogether. Could use a `filtermap/2` maybe and then if TaskInfo is `#{}` return `false`, otherwise return `{true, TaskInfo}` http://erlang.org/doc/man/lists.html#filtermap-2